packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -117,7 +118,14 @@ const Redirect = ({
             break;
         }
         case 'payment_withdraw': {
-            history.push(routes.cashier_withdrawal);
+            if (loginid.toLowerCase().includes('w')) {
```

Comment:
@heorhi-deriv in my opinion, checks only by 1 char sounds as not a good idea, I prefer to use `startsWith`, but, as I said before, you can ignore it :)